src/sentry/tagstore/snuba/backend.py

```diff
@@ -74,35 +69,3 @@ def get_group_ids_for_users(self, project_ids, event_users, limit=100):
 
     def get_groups_user_counts(self, project_id, group_ids, environment_id):
         pass
-
-    # Search
-    def get_group_ids_for_search_filter(self, project_id, environment_id, tags):
```

Comment:
This at the wrong abstraction level now, and is superseded by the `snuba` branch in search. (The same code basically moved though)

Reply:
This might be worth keeping around just to avoid having to deal with errors when doing comparison testing? (Or we can filter it out on the other end — it's not really a big deal.)

Reply:
Ya, moving the code back there so it can be used regardless of which search service is in use.